src/components/app-navigator/Header.tsx

```diff
@@ -0,0 +1,74 @@
+import { useCallback } from 'react'
+
+import { View } from 'react-native'
+
+import AudiusLogo from 'app/assets/images/audiusLogoHorizontal.svg'
+import IconNotification from 'app/assets/images/iconNotification.svg'
+import IconSearch from 'app/assets/images/iconSearch.svg'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { IconButton } from '../core/IconButton'
+
+const useStyles = makeStyles(({ palette, typography, spacing }) => ({
+  header: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    zIndex: 15,
+    width: '100%'
+  },
+  iconNotification: {
+    height: 28,
+    width: 28
+  },
+  audiusLogo: {
+    height: 24,
+    width: 93
+  },
+  iconSearch: {
+    height: 18,
+    width: 18
+  }
+}))
+
+type HeaderProps = { children: string }
+
+export const Header = ({ children }: HeaderProps) => {
+  const styles = useStyles()
+  const { neutralLight4 } = useThemeColors()
+  const navigation = useNavigation()
+
+  const handlePressHome = useCallback(() => {
+    navigation.navigate({
+      native: { screen: 'trending', params: undefined },
+      web: { route: 'trending' }
+    })
+  }, [navigation])
+
+  if (children === 'profile') {
```

Comment:
I'm not super familiar with `Stack.Navigator`, how does `children` get passed through here?

Reply:
yeah it's quite weird, so basically the "title" of each screen, which in our case is the "name" prop of each screen, gets passed as the "children" prop in the "headerTitle" prop

Reply:
ahh gotcha, I was looking at the docs for `header` not `headerTitle`. Any thoughts on using one over the other?

Reply:
would be down for `header` where is that in the docs?

Reply:
was reading stuff here: https://reactnavigation.org/docs/stack-navigator#header-related-options

Reply:
ohh this is exactly what we need, much better that the other prop, great find.